.github/workflows/unit-tests.yml

```diff
@@ -0,0 +1,76 @@
+name: unit-tests
+
+on:
+  pull_request:
+  push:
+    branches:
+      - "master"
+      - "open-release/*.master"
+  
+jobs:
+  run-tests:
+    if: (github.repository != 'openedx/edx-platform' && github.repository != 'edx/edx-platform-private') || (github.repository == 'openedx/edx-platform' && (startsWith(github.base_ref, 'open-release') == true))
+    runs-on: [ ubuntu-20.04 ]
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: [3.8]
+        django-version:
+          - "pinned"
+    
+    name: section-to-course-gh-hosted-python-${{ matrix.python-version }},django-${{ matrix.django-version }}
+    steps:
+      - name: Checkout edx-platform repository
+        uses: actions/checkout@v3
+        with:
+          repository: openedx/edx-platform
+          ref: ${{ github.base_ref }}
```

Comment:
There are two versions of the platform we want to test against-- our own `opencraft-release/nutmeg.2` branch, and upstream master. Eventually, we'll drop support for Nutmeg and test against Palm instead. But the basic idea remains: We'll always want to test against two versions of the platform-- upstream master will always be needed, but we can update this file when we need to change the other one.

Reply:
I believe we should add an `edx-platform-version` variable to the `matrix` to generate tests dynamically.

Reply:
Added edx-platform-version for master & nutmeg.2 branch.